code/datums/components/footstep.dm

```diff
@@ -98,7 +98,7 @@
 			//Sound of wearing shoes always plays, special movement sound
 			// IE (server motors wont play bare footed.)
 			if(H.dna.species.special_step_sounds)
-				playsound(T, pick(H.dna.species.special_step_sounds), 50, TRUE)
+				playsound(LM, pick(H.dna.species.special_step_sounds), 80, TRUE, self_volume = 40)
```

Comment:
this is snowflake for IPCs, make a var on species for step sound volume and override it on IPCs

Reply:
This is how the special step sounds work. Making a var for self step sound volume and then setting it to 40 on every race would just be redundant

Reply:
This sound is always annoying no matter what since it plays regardless of what you are walking on, it plays alongside the regular footstep sounds. This is used by golems and IPCs to play their movement sound always which would always be quieter

Reply:
The perceived volume of the sound file itself can vary, so we should make it possible per species rather than globally hardcoding it. You don't have to set it to 40 per species since you can just make it a species var that defaults to 40

Reply:
You can't control the actual volume of the footstep sounds themselves so I dont see why you would need to be able to control the volume of the footstep sound for the user.